src/components/search/sort.vue

```diff
@@ -1,41 +1,46 @@
 <template>
-  <SortDropdown
-    v-if="$x.totalResults"
-    class="x-dropdown--l"
-    :items="sortValues"
-    :animation="animation"
-  >
-    <template #toggle="{ item }">
-      <span>{{ $t('sort.label') }}</span>
-      <!--TODO: remove the `x-text` class when this task done:
-      https://searchbroker.atlassian.net/browse/EX-4196-->
-      <span class="x-text x-dropdown__toggle-selected-value x-list__item--expand">
-        {{ item || 'Default' }}
-      </span>
-      <ChevronTinyDownIcon />
+  <CustomHeaderTogglePanel :data-test="'sort'">
```

Comment:
You have to add here a gap of 16px:

![image](https://user-images.githubusercontent.com/6597815/175308633-f4793e00-cf44-48c3-bbbc-da3946e2d687.png)